performance/collector/es.py

```diff
@@ -0,0 +1,89 @@
+
+import json
+import requests
+import six
+import os
+
+
+
+from six.moves.urllib import parse as urlparse
+from datetime import datetime
+
+
+from rally import api
+from rally.common import logging
+from rally import exceptions
+from rally.task import exporter
+
+LOG = logging.getLogger(__name__)
+
+
+@exporter.configure(name="es")
+class ESExporter(exporter.Exporter):
+
+    def validate(self):
+        """Validate connection string.
+
+        The format of connection string in ElasticSearch plugin is
+            es:///<path>.<region>
+        """
+        print(self.region)
```

Comment:
- remove all debug statements (print-s from code) 

- if you would like to provide some information use LOG.info or LOG.debug